tests/unit/models/reddit/test_subreddit.py

```diff
@@ -61,6 +64,32 @@ def test_hash(self):
         assert hash(subreddit2) != hash(subreddit3)
         assert hash(subreddit1) != hash(subreddit3)
 
+    @mock.patch(
+        "praw.Reddit.post",
+        return_value={"json": {"data": {"websocket_url": ""}}},
+    )
+    @mock.patch("praw.models.Subreddit._upload_media", return_value="")
+    @mock.patch("websocket.create_connection")
+    def test_invalid_media(self, connection_mock, __, _):
```

Comment:
Please don't use `__` as a variable name. Rather consider using `_mock_upload_media` and `_mock_post` respectively.

Reply:
I don't need to actually do anything with the unnamed mocks, plus all of the `time.sleep` mocks use underscores.

Reply:
Right.  A single underscore as a variable name is fine if you don't use it. If you have more than one such unused variables, then name them uniquely with a single prefix underscore.